dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/validation/field/validators/OrcidFieldValidatorTest.java

```diff
@@ -75,9 +75,9 @@ public void validateValue__non_matching_identifier_type() {
         FieldValidationResult result = validator.validateValue(isni, identifier, params, Collections.emptyMap());
 
         // then
-        assertThat(result.isOk()).isTrue();
-        assertThat(result.getField()).isNull();
-        assertThat(result.getMessage()).isEmpty();
+        assertThat(result.isOk()).isFalse();
```

Comment:
Why should this be false? We don't know whether the ISNI number is valid or not.

Reply:
But there is no mismatch, the ORCID validator should only validate identifiers of type ORCID. It's explained in the javadoc: https://github.com/CeON/dataverse/pull/2583/files#diff-8be394bff9bf9e86e56425c19dfb2eb06a1835f0d579bca8a3ac62384e13d24dR17

Identifiers of type ISNI would be validated by an isni validator, if we had one :)

If the orcid validator would fail the validation for any other scheme, the user wouldn't be able to use them.

Reply:
te test is what it is.
it seems that it doesn't test a valid scenario, so meybe I'll remove it?

Reply:
fiexd - reverted changes